### ISSUE TITLE
Long alt text for macro images can cause the hotbar to render strangely

### ISSUE THREAD

**Issue by Fyorl** [foundryvtt/foundryvtt#8768]
Originally reported by wonko#5436 https://discord.com/channels/170995199584108546/1065764070680186930/1065800335446245406

**ALL MODULES DISABLED?** true  
**OS, Hosting, Browser (if applicable):** Chrome 109.0  
**Short Description of bug:** A macro with long name that is missing the image shows the alt text. The alt text overflows and shifts the whole UI upwards, hiding the top of the sidebar.  
**Simple steps to reproduce the bug:** Macro with long name and non-resolving image URL. Reproducing this by changing the image path will require reload or sidebar tab change to trigger. Doesn't seem to happen in V10.  
**Screenshots and/or console errors:**

![image](https://user-images.githubusercontent.com/207433/213720414-9ff168cb-c045-4f9e-ae3d-a7fa7074da48.png)

**Comment by aaclayton** [comment]
Not verified in 11.293. Closing as nonrepro:


![Image](https://user-images.githubusercontent.com/2468703/222552345-29d61967-b565-4d24-82a9-86042a7241fc.png)

**Comment by Norc** [comment]
After gathering feedback from the helpers / moderators, I have been able to come up with steps to reliably repro. I think that the missing ingredients were:

1. There needs to be at least one chat message in the chat log
2. The long macro name needs a few spaces in it so it can break into several lines.

 **Test environment**: Electron app, Foundry 11.308, dnd5e 2.3.1. Also verfied in PF2E 5.3.2 with 11.308. All modules disabled.

**Steps to recreate**:

1. Create new world
2. Click the macro hotbar to create a new macro. Click Save.
3. Paste and run the following script into the console (for conveniently breaking the image link:
```
let m = game.macros.contents[0];
const testImg = 'foo.png';
const testLongName = 
  '1234567891 1234567892 1234567893 1234567894'
const testData = {img: testImg,
               name: testLongName,
               };                
await m.update(testData);
```
4. Press f5
5. Type and send any new chat message

**Current results**: The extra rows of macro name text next to the broken image icon push the hotbar up, also pushing up the entire Foundry UI, resulting in the "chop top" of the sidebar menus:
![image](https://github.com/foundryvtt/foundryvtt/assets/64667579/363b60b1-b3f1-4e55-b404-8c28a1ae12ae)

**Expected results**:  The broken image caption of the macro name is handled in such a way that it does not push the entire Foundry UI up. Perhaps it could truncate it, employ wrapping, or simply not be displayed at all.

**Testing notes**:  
1. There needs to be at least one chat message in the chat log for the issue to trigger. The "Getting Started" message fills this role in these test steps, otherwise you will need to manually type it.
2. It only triggers on a refresh, such as F5 or typing a new chat message. Changing the macro page so that the problem macro is visible does not trigger the issue, nor does it happen at first creation.
3. The long macro name **must** have spaces, or the name will not wrap. it is the number of lines of text that is important here, not the total character count.  
4. Changing the name to something even longer, such as `1234567891 1234567892 1234567893 1234567894 1234567895 1234567896 1234567897 1234567898 123456789 912345678910 12345678911 12345678912 12345678913 12345678914 12345678915 12345678916 12345678917 12345678918 12345678919 12345678920` makes the effect even more pronounced.

**Comment by aaclayton** [comment]
I don't see macro ALT text in the above image example. Are you sure it's the same issue?

**Comment by Norc** [comment]
No, I don't mean to imply that it is, but it's interesting that the same symptom seems to be able to (rarely) caused without broken macro images or long alt text involved at all. 

I was just including this as another data point... it's leading me to suspect that there is an underlying issue with the Foundry CSS that is triggering this UI push-up behavior, and identifying and resolving that (if it exists) might be more useful than just, say, changing the way that alt text is displayed for broken macro images.

Sorry if this isn't helpful!
